src/main/scala/scorex/core/network/peer/PeerInfo.scala

```diff
@@ -9,11 +9,11 @@ import scorex.core.network.{ConnectionDirection, PeerSpec}
   * Information about peer to be stored in PeerDatabase
   *
   * @param peerSpec       - general information about the peer
-  * @param lastSeen       - timestamp when this peer was last seen in the network
+  * @param lastHandshake       - timestamp when this peer was last seen in the network
```

Comment:
timestamp when last handshake was done (last seen sounds ambiguous, i.e. can be considered as time when we got a message from a peer last time)

Reply:
Done